README.md

```diff
@@ -119,21 +99,23 @@ If no arguments are specified, Shishito, by default, searches for BROWSERSTACK c
 * changes to variables should be maintained in VCS; so that configuration can be reused for automated test execution
 
 ```
+# modules
+test_platform=web
+test_environment=local
+
+# test dir
+test_directory=tests
+
 # General
 base_url=http://www.google.com
-driver=Firefox
-
-# Remote driver
-remote_hub=http://localhost:4444/wd/hub
-browser_version=34.0
-platform=WINDOWS
+environment_configuration=Chrome
 ```
 
+* *test_platform* - on which platform run tests (web, mobile)
+* *test_environment* - in which environment run tests (local, browserstack, appium)
+* *test_directory* - in which directory lookup for tests
 * *base_url* - url that will be loaded by default upon start of each test
-* *driver* - name of driver used. For Browserstack use "BrowserStack"
-* *remote_hub* - remote driver hub. If specified & BrowserStack is not used, tests will be run on remote driver (not on local browsers)
-* *browser_version* - version of browser. Used for remote driver (Selenium Grid, not Browserstack)
-* *platform* - OS platform. Used for remote driver (Selenium Grid, not Browserstack)
+* *environment_configuration* - which configuration use from <environment>.properties file (used when tests are run without runner)
```

Comment:
Please add: *remote_driver_url* - remote driver hub. Selenium server needs to be running on this url.